test/functional/legacy/044_099_regexp_multibyte_magic_spec.lua

```diff
@@ -85,6 +86,11 @@ local function run_test_with_regexpengine(regexpengine)
   execute([[let @w=':%s#comb[i]nations#œ̄ṣ́m̥̄ᾱ̆́#g']])
   execute('@w')
 
+  -- Line l. Ex command ":s/ \?/ /g" should NOT split multi-byte characters
+  -- into bytes (fixed by vim-7.3.192).
+  feed('/^l')
+  feed('s/ \?/ /g)
```

Comment:
missing closing quote, and the `\?` is escaped by lua. Changed to this:

    execute([[s/ \?/ /g]])